models/issue_dependency.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"code.gitea.io/gitea/modules/setting"
+	"time"
```

Comment:
import order

Reply:
FIxed.

Reply:
See https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#styleguide

Reply:
@kolaente still a missing one.

Reply:
Github didn't got it...
But you can see the change when directly viewing it in my repository: https://github.com/kolaente/gitea/blob/master/models/issue_dependency.go